### PR TITLE
Layout/DOM edits (WIP)

### PR DESCRIPTION
**Pull request by rhrynyszyn**
Touched up CSS layouts a bit, began work on the DOM debugging page... I think I am fairly close with my launguage, but may have some inaccuracies.

### CODE REVIEW

File: public/lessons/www/css_layout.md [hunk 1]
```diff
@@ -50,10 +50,10 @@ Inherent display properties of commonly-used HTML elements.
...
-	- Elements moved around with the properties top, left, right, bottom. ``top:20px;`` will move a relatively positioned element 20 pixels from its natural postioning.  
-2. Absolute - This is positioned relative to its parent or ancestor (closest ancestor that is relatively positioned). 
-	- Any element that is positioned absolutely, will be placed (using the top/left/right/bottom CSS properties) specifically within the parent, irrespective of other sibling elements.
-3. Fixed - Position an element is relation to its viewport (browser window).
+	- Elements are moved around with the properties top, left, right, bottom. ``top:20px;`` will move a relatively positioned element 20 pixels from its natural postioning.  
+2. Absolute - This is positioned relative to its closest ancestor that is relatively positioned. 
+	- Any element that is positioned absolutely, will be placed (using the top/left/right/bottom CSS properties) specifically within the parent, irrespective of other sibling elements. ```top:20px; left:20px;``` will place an element 20 pixels from the top and 20 pixels from the left of its ancestor who has the property ```position:relative;```
```
Comment (alexch) [inline public/lessons/www/css_layout.md]: just use one backtick for inline code; triple backticks are just for multiline code blocks

**Comment by alexch** [comment]
@rhrynyszyn  is this PR still active? can we merge it or close it?

**Comment by rhrynyszyn** [comment]
This can be closed. I'll update to single ticks in other open PR

**Comment by Dangeranger** [comment]
This PR is out of date and the changes have been committed in another PR/Commit.

**Pull request closed without merging**